_ml-commons-plugin/conversational-search.md

```diff
@@ -335,6 +348,19 @@ PUT /_search/pipeline/<pipeline_name>
 
 You can customize `context_field_list` in your RAG pipeline to send any fields that exist in your documents to the LLM.
 
+### RAG parameter options
+
+Use the following options when setting up a RAG pipeline under the `retrieval_augmented_generation` argument:
+
+Parameter | Required | Description
+:--- | :--- | :---
+`tag` | Yes | A tag to help identify the pipeline.
+`description` | Yes | A description of the pipeline.
+`model_id` | Yes | The ID of the model used in the pipeline.
+`context_field_list` | Yes | The list of fields in document sources that the pipeline uses as context for the RAG. For more information, see [Context Field List](#context-field-list).
+`system_prompt` | No | The message sent to the LLM with a `system` role. This is the message the user sees when the LLM receives an interaction.
```

Comment:
So if it's not required, what does the user see if system prompt is not provided?

Reply:
Whatever prompt was provided in the interaction.

Reply:
This is an override for the interactions set in the document or in Conversation API.